django/__init__.py

```diff
@@ -5,12 +5,26 @@
 __version__ = get_version(VERSION)
 
 
+# Flag also used to prevent multiple setup() calls
+is_ready = False
+
+
 def setup(set_prefix=True):
     """
     Configure the settings (this happens as a side effect of accessing the
     first setting), configure logging and populate the app registry.
     Set the thread-local urlresolvers script prefix if `set_prefix` is True.
+
+    This function is idempotent (calling it several times changes nothing), but
+    not reentrant (recursively calling it would have an unexpected result).
+
+    Returns True iff actual operations occurred.
```

Comment:
I think the current wording is a bit hard to follow:
* "iff" is a rare, more academic word, that won't translate well
* "actual operations occurred" is a bit vague
* The case where "actual operations didn't occur" is not clearly specified - will `None` be returned.

How about "Returns a boolean indicating whether setup occurred for the first time" ?

Reply:
Ok changed